crowbar_framework/app/models/crowbar_service.rb

```diff
@@ -62,8 +62,8 @@ def transition(inst, name, state)
         return [404, "Node not found"]
       end
 
-      if state == "readying"
-        transition_to_readying inst, name, state, node
+      if state == "ready" && !node.admin?
```

Comment:
It probably makes sense to explain at least in the commit message (and possibly with a comment here) that we need ohai data, so at least one chef-client run after the install to claim raid disks.

That being said... This means that we try to claim raid disks every time we move to ready, which also means every time we are done with applying a proposal. Not necessarily a big deal, though.

But wouldn't it make more sense to move the claiming to a cookbook (provisioner-base, I guess)? It somehow feels wrong to do it on the webui side.